workflow-templates/shellcheck.yml

```diff
@@ -0,0 +1,29 @@
+# Run ShellCheck on PRs and $default-branch
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+on:
+  pull_request:
+  push:
+    branches:
+    - $default-branch
+
+permissions:
+  contents: read
+
+jobs:
+  shellcheck:
+    strategy:
+      matrix:
+        os: ["ubuntu", "macos"]
+    uses: openedx/.github/.github/workflows/shellcheck.yml@master
+    with:
+      # For details on the meaning of each of these arguments, see:
+      # https://github.com/openedx/.github/blob/master/.github/workflows/shellcheck.yml
+      exclude-patterns: "./node_modules/*"
```

Comment:
```suggestion
      # We exclude `./node_modules/*` by default because we want people to easily be able to
      # copy and run the command locally.  Local copies of most of our services have a `./node_modules` 
      # directory that we want to ignore.
      exclude-patterns: "./node_modules/*"
```